pkg/harvester/components/FilterVMSchedule.vue

```diff
@@ -0,0 +1,119 @@
+<script>
+import { RadioGroup } from '@components/Form/Radio';
+
+export default {
+  name: 'HarvesterFilterVMSchedule',
+
+  components: { RadioGroup },
+
+  props: {
+    rows: {
+      type:     Array,
+      required: true,
+    },
+  },
+
+  data() {
+    return { selected: '' };
+  },
+
+  computed: {
+    scheduleOptions() {
+      const options = this.rows.map(r => r.sourceSchedule).filter(r => r);
+
+      return Array.from(new Set(options));
+    },
+  },
+
+  methods: {
+    onSelect(selected) {
+      this.selected = selected;
+      this.filterRows();
+    },
+
+    remove() {
+      this.selected = '';
+      this.filterRows();
+    },
+
+    filterRows() {
+      if (!this.selected) {
+        this.$emit('change-rows', this.rows);
+
+        return;
+      }
+      const filteredRows = this.rows.filter(row => row.sourceSchedule === this.selected);
+
+      this.$emit('change-rows', filteredRows, this.selected);
+    }
+  },
+
+  watch: {
+    rows: {
+      deep:      true,
+      immediate: false,
+      handler() {
+        this.filterRows();
+      }
+    }
+  }
+};
+</script>
+
+<template>
+  <div class="vm-schedule-filter">
+    <template>
+      <span v-if="selected" class="banner-item bg-warning">
+        {{ t('harvester.tableHeaders.vmSchedule') }}{{ selected ? ` = ${selected}`: '' }}<i class="icon icon-close ml-5" @click="remove" />
+      </span>
+    </template>
+
+    <v-popover
+      trigger="click"
```

Comment:
There is a small empty box showing up when clicking the filter, but there are not options to show:

![image](https://github.com/user-attachments/assets/6d7b10df-1fbb-4993-a84d-98a4532cc125)

To fix it:

```suggestion
     :trigger="scheduleOptions.length ? 'click' : 'manual'"
```